eng/targets/Helix.Common.props

```diff
@@ -23,7 +23,7 @@
     <HelixAvailableTargetQueue Include="Debian.8.Amd64.Open" Platform="Linux" />
     <HelixAvailableTargetQueue Include="Debian.9.Amd64.Open" Platform="Linux" />
     <HelixAvailableTargetQueue Include="Redhat.7.Amd64.Open" Platform="Linux" />
-    <HelixAvailableTargetQueue Include="Fedora.28.Amd64.Open" Platform="Linux" />
+    <HelixAvailableTargetQueue Include="(Fedora.28.Amd64.Open)Ubuntu.1604.Amd64.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:fedora-28-helix-09ca40b-20190508143249" Platform="Linux" />
```

Comment:
What does the paren syntax mean?

Reply:
The parenthesis part is queue spoofing for reporting purposes only; if you don't need it, you can leave it out.  If the run is viewed on Mission Control without this, it'd report as the machines it actually ran on, Ubuntu.1604.Amd64.Open, which is confusing once you're using a bunch of them.